reana/reana_dev/cluster.py

```diff
@@ -376,11 +376,35 @@ def cluster_unpause():
     run_command(cmd, "reana")
 
 
+@click.option(
+    "-m",
+    "--mount",
+    "mounts",
+    multiple=True,
+    help="Which local path directories are to be deleted? [local_path:cluster_node_path]",
+)
 @cluster_commands.command(name="cluster-delete")
-def cluster_delete():
-    """Delete REANA cluster."""
-    cmd = "kind delete cluster"
-    run_command(cmd, "reana")
+def cluster_delete(mounts):  # noqa: D301
+    """Delete REANA cluster.
+
+    \b
+    Example:
+       $ reana-dev cluster-delete -m /var/reana:/var/reana
+    """
+    cmds = []
+    # delete cluster
+    cmds.append("kind delete cluster")
+    # remove only local paths which starts with /var/reana for safety
+    for mount in mounts:
+        local_path = mount.split(":")[0]
```

Comment:
Actually I realised we should test `cluster_node_path` for starting with '/var/reana', because on some systems, users may need to do `cluster-create -m /opt/mybigdisk/reana:/var/reana` in order to mount a big local disk from a non-standard location into cluster's `/var/reana`.  I'll amend this while merging...